llama_stack/apis/files/files.py

```diff
@@ -77,6 +77,38 @@ class ListFileResponse(BaseModel):
     data: List[FileResponse]
 
 
+@json_schema_type
+class BucketRegistrationResponse(BaseModel):
+    """
+    Response after registering a storage location.
+
+    :param bucket: The registered storage location URI (e.g., "s3://my-bucket" or "file:///data")
+    :param created_at: Timestamp of registration
+    :param status: Current status of the storage location
+    """
+
+    bucket: str
+    created_at: int
+    status: str
```

Comment:
Which statuses do you think of here? Should this be an enum?

Reply:
I was thinking whether it's "active" or not but since we are only registering it's overkill. Good catch.